gcloud/logging/config_service_v2_api.py

```diff
@@ -14,64 +14,123 @@
 #
 # EDITING INSTRUCTIONS
 # This file was generated from the file
-# https://github.com/google/googleapis/blob/7710ead495227e80a0f06ceb66bdf3238d926f77/google/logging/v2/logging_config.proto,
+# https://github.com/google/googleapis/blob/master/google/logging/v2/logging_config.proto,
```

Comment:
Please file an issue.

The actual java language repo with the code generator code in it is not public yet, but we're tracking python codegen issues using a label for now https://github.com/googleapis/gax-python/labels/code-generation.